bin/test

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-NAMESPACE=staging
+NAMESPACE=production
```

Comment:
This should stick to staging, we are not ready for production yet

Reply:
okay